cfbs.json

```diff
@@ -883,11 +883,51 @@
       "tags": ["security", "compliance"],
       "repo": "https://github.com/nickanderson/cfengine-lynis",
       "by": "https://github.com/nickanderson",
-      "version": "3.1.1",
-      "commit": "70bdf7be29c890d3bf162a8fe993e8d12eb61388",
+      "version": "3.1.1-2",
+      "commit": "4db6d515c45873a9f3fa7ce78ece593027464b64",
       "steps": [
         "copy policy/main.cf services/lynis/main.cf",
-        "json cfbs/def.json def.json"
+        "json cfbs/def.json def.json",
+        "input ./input.json def.json"
+      ],
+      "input": [
+        {
+          "type": "string",
+          "namespace": "lynis",
+          "bundle": "globals",
+          "variable": "version",
+          "label": "Lynis version",
+          "question": "What version of Lynis should be used?",
+          "default": "3.1.1"
+        },
+        {
+          "type": "string",
+          "namespace": "lynis",
+          "bundle": "globals",
+          "variable": "tar_url",
+          "label": "Tarball url",
+          "question": "Where should clients download lynis from (http/https)",
+          "default": "https://downloads.cisofy.com/lynis/lynis-$(version).tar.gz",
+          "comment": "Some may want to self host the tarball within their own infrastructure. This provides an easy way to do that. Note, the archive should be named for the version of Lynis selected."
+        },
+        {
+          "type": "string",
+          "namespace": "lynis",
+          "bundle": "globals",
+          "variable": "archive_hash",
+          "label": "Hash of the tarball",
+          "question": "What is the hash of the tarball?",
+          "default": "d72f4ee7325816bb8dbfcf31eb104207b9fe58a2493c2a875373746a71284cc3"
```

Comment:
and here's the reason why "latest" would be tough with the version. If you supported that then you would have to pull the hash from the website and that's not as secure.